CONTRIBUTING.md

```diff
@@ -135,10 +135,10 @@ We have three main branches in our repo:
 ### Release process
 
 1. Ensure all code to be released is merged into `dev`.
-2. Create a release branch locally based on `master`.
-3. Merge commits to be relaased into the release branch. Avoid cherry-picking changes so that `master` always remains a subset of `dev`.
-4. Update the version number in `package.json` according to [semantic versioning](https://semver.org/). 
+2. Make a PR into `dev` to update the version number in `package.json` according to [semantic versioning](https://semver.org/), 
    - Note: if the only changes being made are to the documentation, do not update the version number.  In this case the GitHub Pages site will be published when `master` is built, but no new version of the plug-in will be published.
-5. Make a PR to merge your release brnach into `master`. The reviewer should make sure all commits are suitable to release and version number has been updated appropriately. 
+3. Create a release branch locally based on `master`.
```

Comment:
I think only the collaborators or admins should manage the `release` branches and the branch should appear on the main repo so that it can allow hot fixes or doc changes just before the release.

Reply:
Hmm, why would you want to make any distinct changes in a release branch and not put them in dev first? They'll need to end up in dev eventually otherwise merging dev -> master will be a pain.